File: Habit.java
public class Habit extends TrackerEvent {
    public Habit(int id, String name) {
        super(id, name);
    }
    @Override
    public String getDetails() {
        return "Habit: " + getName();
    }
    @Override
    public String toString() {
        return getName();
    }
}

File: Habit.py
from tracker_event import TrackerEvent

class Habit(TrackerEvent):
    def __init__(self, id: int, name: str):
        super().__init__(id, name)

    def get_details(self) -> str:
        return f"Habit: {self.get_name()}"

    def __str__(self) -> str:
        return self.get_name()